pkg/log/url.go

```diff
@@ -41,7 +41,7 @@ func (cl *URLLogger) WriteURLsLogToJSON() error {
 	if len(cl.buf.data) == 0 {
 		return nil
 	}
-	file, err := os.OpenFile(urlLogFileName, os.O_CREATE|os.O_RDWR, 0600)
+	file, err := os.OpenFile(urlLogFileName, os.O_CREATE|os.O_RDWR, 0666)
```

Comment:
Is `0666` necessary? In other cases we use `0644`.

Reply:
But is it always runs by the root user?

Reply:
Usually we run with user `1000`. But we can also go with `0666` to be sure.